shell/initialize/App.vue

```diff
@@ -60,18 +55,16 @@ export default {
     },
   },
 
-  components: { NuxtLoading, NuxtError }
+  components: { NuxtLoading }
 };
 </script>
 <template>
   <div id="__nuxt">
     <NuxtLoading ref="loading" />
     <div
       id="__layout"
-      :key="showErrorPage"
     >
-      <NuxtError v-if="showErrorPage" />
```

Comment:
No where in the file can showErrorPage ever be set to true and I found no references in any other files when doing a global search on the code base.

Since it can never be true the NuxtError can never be shown so I removed it.